addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/kapp-config.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ako-kapp-config
+  labels:
+    kapp.k14s.io/config: ""
+data:
+  config.yml: |
+    apiVersion: kapp.k14s.io/v1alpha1
+    kind: Config
+    rebaseRules:
+    - paths:
+      - [annotations]
```

Comment:
I don't think this path is correct?

Shouldn't it be `[metadata,annotations]`

Reply:
oh wow, great catch